### PR TITLE
Change to-markdown with turndown

### PR DESCRIPTION
**Pull request by roose**
to-markdown renamed to turndown
Add gfm option

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,89 +2,96 @@
```
Comment (fabianmoronzirfas) [inline index.js]: Hm. By using your linter settings yo removed all the `;` but I really like them :-(

File: index.js [hunk 2]
```diff
@@ -2,89 +2,96 @@
...
+}
 
-if(program.clipboard !== undefined) {
-  data = clipboardy.readSync();
-  data = toMarkdown(data);
-  clipboardy.writeSync(data);
-  process.exit(0);
+if (program.clipboard !== undefined) {
+  data = clipboardy.readSync()
+  data = turndownService.turndown(data)
```
Comment (fabianmoronzirfas) [inline index.js]: This is an actual change

File: index.js [hunk 3]
```diff
@@ -2,89 +2,96 @@
...
+} else {
+  data = clipboardy.readSync()
 }
 
-if(program.output !== undefined) {
-  fs.writeFile(outPath, toMarkdown(data), 'utf8', (error, md) => {
-    if(error) {
-      throw error;
+if (program.output !== undefined) {
+  fs.writeFile(outPath, turndownService.turndown(data), 'utf8', (error, md) => {
```
Comment (fabianmoronzirfas) [inline index.js]: This too

File: index.js [hunk 4]
```diff
@@ -2,89 +2,96 @@
...
+      throw error
     }
-    console.log(`wrote to ${outPath}`);
-  });
-}else{
-  console.log(toMarkdown(data));
+    console.log(`wrote to ${outPath}`)
+  })
+} else {
+  console.log(turndownService.turndown(data))
```
Comment (fabianmoronzirfas) [inline index.js]: And this one

File: package.json [hunk 5]
```diff
@@ -9,17 +9,24 @@
...
   "preferGlobal": true,
   "bin": {
     "html2md": "./index.js"
   },
   "repository": "fabianmoronzirfas/to-markdown-cli",
   "devDependencies": {
-    "cz-conventional-changelog": "^2.0.0"
+    "cz-conventional-changelog": "^2.1.0",
+    "eslint": "^5.5.0",
+    "eslint-config-standard": "^12.0.0",
```
Comment (fabianmoronzirfas) [inline package.json]: I'm actually not happy with these plugins. Are they needed? 
I can give a example of my usual eslint config if you like.

File: index.js (hunk 1 above)
Comment (roose) [reply index.js]: Check please

**Comment by fabianmoronzirfas** [comment]
Awesome! Thank you

**Pull request merged**